PIL/Image.py

```diff
@@ -504,26 +535,30 @@ class Image(object):
     _close_exclusive_fp_after_loading = True
 
     def __init__(self):
+        # type: () -> None
         # FIXME: take "new" parameters / other image?
         # FIXME: turn mode and size into delegating properties?
         self.im = None
         self.mode = ""
         self.size = (0, 0)
-        self.palette = None
-        self.info = {}
+        self.palette = None  # type: Optional[ImagePalette.ImagePalette]
+        self.info = {}  # type: Dict[Text, Any]
```

Comment:
Info can be almost anything. It's pulled from the metadata in images, so bytes, string, tuples of int/float/?, float, ints are certainly possible. You can pretty much fit a ham sandwich in tiff metadata, and that's the source for a lot of this.